rest/backend_server/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "bff"
+name = "rest"
+version = "0.1.0"
+distribution = "2201.8.2"
+
+[build-options]
```

Comment:
We can remove this if not needed.

Reply:
Those are added while creating a new Ballerina project, 
Yes we can remove that.

Reply:
updated